java/README.md

```diff
@@ -55,3 +55,23 @@
      human-readable.
 
 9. Update the code to insert a document with the regular client. What happens?
+
+## KMIP
+
+**KMIP support in this guide is to be considered "best effort"**.
+
+This guide uses the `direnv` tool to manage environment variables. 
```

Comment:
comment: I think it would be helpful to include a link to the direnv git repo (which I think tells people to use brew)

Reply:
After looking at the [direnv git repo](https://github.com/direnv/direnv) and [direnv.net](https://direnv.net/), it seems like direnv.net may be more helpful as it seems to contain the content of the repos readme file without listing out the source code files of direnv. I've added a link to direnv.net, let me know if this is alright or if you disagree with this decision.